theme/mixins/backdrop.less

```diff
@@ -1,11 +1,15 @@
-.backdrop() {
+.transparentBackdrop() {
   position: fixed;
   top: -999px;
   left: -999px;
   right: -999px;
   bottom: -9999px;
   background-color: @sk-black;
```

Comment:
The `background-color` should be in the `shadedBackdrop` mixin too, arguably along with the `transition` as well.